specifyweb/frontend/js_src/lib/components/DataModel/specifyField.ts

```diff
@@ -250,9 +254,20 @@ export class LiteralField extends FieldBase {
 
   public readonly isRelationship: false = false;
 
+  // Indicates white space should not be ignored in the field
+  // eslint-disable-next-line functional/prefer-readonly-type
+  public whiteSpaceSensitive: boolean;
```

Comment:
add `readonly`?

Reply:
Fixed in https://github.com/specify/specify7/pull/5076/commits/baeb3725f17fed8f57bcd817ab31a2724c05fcc4